PWGHF/DataModel/ReducedDataModel.h

```diff
@@ -0,0 +1,247 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file ReducedDataModel.h
+/// \brief Header file with definition of methods and tables
+//  used to fold (unfold) track and primary vertex information by writing (reading) AO2Ds
+/// \note
+///
+/// \author Alexandre Bigot <alexandre.bigot@cern.ch>, IPHC Strasbourg
+
+#ifndef PWGHF_DATAMODEL_REDUCEDDATAMODEL_H_
+#define PWGHF_DATAMODEL_REDUCEDDATAMODEL_H_
+
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoA.h"
+#include "Common/Core/RecoDecay.h"
+#include "Common/DataModel/PIDResponse.h"
+#include "ReconstructionDataFormats/Track.h"
+#include "ReconstructionDataFormats/Vertex.h"
+#include "PWGHF/DataModel/CandidateReconstructionTables.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+namespace o2
+{
+namespace aod
+{
+namespace hf_reduced_collision
+{
+DECLARE_SOA_COLUMN(Bz, bz, float); //! Magnetic field in z-direction
+// keep track of the number of studied events (for normalization purposes)
+DECLARE_SOA_COLUMN(OriginalAODSize, originalAODSize, int); //! Size of COLLISION table processed
+
+} // namespace hf_reduced_collision
+
+DECLARE_SOA_TABLE(HfReducedCollisions, "AOD", "HFREDCOLLISION", //! Table with collision for reduced workflow
+                  track::CollisionId,                           // FIXME : soa::Index<> does not work event with DECLARE_EQUIVALENT_FOR_INDEX(aod::Collisions, aod::HfReducedCollisions)
```

Comment:
We could consider to define an index for our reduced collisions that does not correspond to the original collision index. However this would require to store as `collisionId` in the pion and D+ tracks the one of the reduced collision and not the one of the original collision.